Docs/sphinx_documentation/source/BuildingAMReX.rst

```diff
@@ -34,6 +34,8 @@ list of important variables.
    | AMREX_HOME | Path to amrex                       | environment |
    +------------+-------------------------------------+-------------+
    | COMP       | gnu, cray, ibm, intel, llvm, or pgi | none        |
+   +------------|-------------------------------------+-------------+
+   | USE_CXXSTD | C++ Standard (11, 14, 17, ...)      | 11          |
```

Comment:
Can we name the variable CXXSTD instead of USE_CXXSTD? We usually reserve the USE_ prefix for boolean variables that either are TRUE or FALSE.

Reply:
absolutely, thanks for the hint! :)